tests/test_replacement.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+from itertools import combinations
+
+from ordering import Ordering
+
+
+class TestOrderingReplacement(TestCase):
+    items_list = [0], [1, 0], [2, 0, 1]
+    absent_list = 1, 2, 3
+
+    def test_basic_replace(self) -> None:
+        ordering = Ordering[int]((1, 0))
+
+        ordering.replace(0, 42)
+        self.assertNotIn(0, ordering)
+        self.assertListEqual(list(ordering), [1, 42])
+        ordering.replace(42, 0)
+        self.assertNotIn(42, ordering)
+        self.assertListEqual(list(ordering), [1, 0])
```

Comment:
A bit of whitespace around each of the `assert` blocks might make this a bit more readable.